# request
bool start
bool clockwise
---
# response
bool completed
string message